pktgen_s/scheduler/pktgen_scheduler.py

```diff
@@ -0,0 +1,366 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
not going to review this very carefully. what really matters is that we now have a way to communicate with your new pktgen server. anybody can then take the protobufs and write their own console/traffic generation scripts